models/aeif_psc_alpha.cpp

```diff
@@ -94,19 +94,20 @@ nest::aeif_psc_alpha_dynamics( double,
   // good compiler will optimize the verbosity away ...
 
   // shorthand for state variables
-  const double& V = std::min( y[ S::V_M ], node.P_.V_peak_ ); // bound V
+  const double& V = std::min( y[ S::V_M ], node.P_.V_peak_ ); // bind V to the
+  // USER DEFINED V_peak_ value in Parameters.
   const double& dI_syn_ex = y[ S::DI_EXC ];
   const double& I_syn_ex = y[ S::I_EXC ];
   const double& dI_syn_in = y[ S::DI_INH ];
   const double& I_syn_in = y[ S::I_INH ];
   const double& w = y[ S::W ];
 
-  const double I_spike =
-    node.P_.Delta_T * std::exp( ( V - node.P_.V_th ) / node.P_.Delta_T );
+  const double I_spike = node.P_.Delta_T == 0. ? 0. : node.P_.g_L
+      * node.P_.Delta_T * std::exp( ( V - node.P_.V_th ) / node.P_.Delta_T );
```

Comment:
For clarity, I would put the entire expression after `:` in parentheses, even though they are not strictly needed in C++.